packages/blade/src/components/Input/TextInput/TextInput.tsx

```diff
@@ -350,6 +363,7 @@ const _TextInput: React.ForwardRefRenderFunction<BladeElementRef, TextInputProps
         autoCompleteSuggestionType,
         autoCapitalize,
       })}
+      size={size}
       {...styledProps}
```

Comment:
can you check if styledProps are working fine on inputs? They were not working for me on storybook args